drivers/can/can_mcp2515.c

```diff
@@ -245,7 +245,7 @@ static void mcp2515_convert_mcp2515frame_to_zcanframe(const u8_t *source,
 	target->rtr = source[MCP2515_FRAME_OFFSET_DLC] & BIT(6) ?
```

Comment:
You limit DLC here

Reply:
General reply here: https://github.com/zephyrproject-rtos/zephyr/pull/20218#discussion_r341544439